sample-apps/spark/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
+#Tue Mar 16 11:14:39 PDT 2021
```

Comment:
Actually we can bump gradle wrapper to a new major version (though I think this is modified by IDEA because it used to suggest me to download full gradle to get better competition etc.

Reply:
yea. it is updated by IDE.